ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -209,6 +228,10 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
 
 RCT_EXPORT_METHOD(cancelScheduledNotification:(NSString *)uniqueId)
 {
+  if (![EXEnvironment sharedEnvironment].isDetached) {
+    uniqueId = [self prefixWithExperience:uniqueId];
```

Comment:
@sjchmiela in other places of the codebase do we use the word "scope" or "prefix" more?

Reply:
I think we could follow naming of `scopedModules` here. An easy refactoring win would be to always pass `uniqueId` through some function named differently, eg. `[self categoryIdForUserId:uniqueId]` (I'm not really a fan of `UserId`, but you get the gist). This function could `if isDetached`.